.github/workflows/drivers-stress-test.yml

```diff
@@ -109,42 +108,6 @@ jobs:
         junit-name: 'be-tests-druid-ee'
         test-args: ":only ${{ inputs.test }} :times ${{ inputs.times }}"
 
-  be-tests-googleanalytics-ee:
-    if: inputs.driver == 'Google Analytics'
-    runs-on: ubuntu-22.04
-    timeout-minutes: 90
-    env:
-      CI: 'true'
-      DRIVERS: googleanalytics
-    steps:
-    - uses: actions/checkout@v4
-    - name: Test Google Analytics driver
-      uses: ./.github/actions/test-driver
-      with:
-        junit-name: 'be-tests-googleanalytics-ee'
-        test-args: ":only ${{ inputs.test }} :times ${{ inputs.times }}"
-
-  be-google-related-drivers-classpath-test:
```

Comment:
this job was to check that Google Analytics and Big Query dependencies didn't conflict with each other so we don't need it anymore